TGC.MonoGame.Samples/Physics/Bepu/Callbacks.cs

```diff
@@ -7,104 +7,185 @@
-
-            //Note that you CAN technically modify the pose in IntegrateVelocity by directly accessing it through the Simulation.Bodies.ActiveSet.Poses, it just requires a little care and isn't directly exposed.
-            //If the PositionFirstTimestepper is being used, then the pose integrator has already integrated the pose.
-            //If the PositionLastTimestepper or SubsteppingTimestepper are in use, the pose has not yet been integrated.
-            //If your pose modification depends on the order of integration, you'll want to take this into account.
-
-            //This is also a handy spot to implement things like position dependent gravity or per-body damping.
-        }
+    /// <summary>
+    ///     Callback for a bundle of bodies being integrated.
+    /// </summary>
+    /// <param name="bodyIndices">Indices of the bodies being integrated in this bundle.</param>
+    /// <param name="position">Current body positions.</param>
+    /// <param name="orientation">Current body orientations.</param>
+    /// <param name="localInertia">Body's current local inertia.</param>
+    /// <param name="integrationMask">
+    ///     Mask indicating which lanes are active in the bundle. Active lanes will contain
+    ///     0xFFFFFFFF, inactive lanes will contain 0.
+    /// </param>
+    /// <param name="workerIndex">Index of the worker thread processing this bundle.</param>
+    /// <param name="dt">Durations to integrate the velocity over. Can vary over lanes.</param>
+    /// <param name="velocity">
+    ///     Velocity of bodies in the bundle. Any changes to lanes which are not active by the
+    ///     integrationMask will be discarded.
+    /// </param>
+    public void IntegrateVelocity(Vector<int> bodyIndices, Vector3Wide position, QuaternionWide orientation,
+        BodyInertiaWide localInertia, Vector<int> integrationMask, int workerIndex, Vector<float> dt,
+        ref BodyVelocityWide velocity)
+    {
+        //This is a handy spot to implement things like position dependent gravity or per-body damping.
+        //This implementation uses a single damping value for all bodies that allows it to be precomputed.
+        //We don't have to check for kinematics; IntegrateVelocityForKinematics returns false, so we'll never see them in this callback.
+        //Note that these are SIMD operations and "Wide" types. There are Vector<float>.Count lanes of execution being evaluated simultaneously.
+        //The types are laid out in array-of-structures-of-arrays (AOSOA) format. That's because this function is frequently called from vectorized contexts within the solver.
+        //Transforming to "array of structures" (AOS) format for the callback and then back to AOSOA would involve a lot of overhead, so instead the callback works on the AOSOA representation directly.
+        velocity.Linear = (velocity.Linear + GravityWideDt) * LinearDampingDt;
+        velocity.Angular = velocity.Angular * AngularDampingDt;
```

Comment:
La sugerencia de parte del linter es utilizar una asignación compuesta, como lo siguiente
```suggestion
        velocity.Angular *= AngularDampingDt;
```

Reply:
Claro el tema que esa clase en realidad esta "tomada" de Bepu medio que da igual esos detalles.